Controllers/CafeApiController.cs

```diff
@@ -0,0 +1,50 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using cafeRecAPI.DTO;
+using cafeRecAPI.Models;
+using cafeRecAPI.Infra;
+using cafeRecAPI.Data;
+using System.Runtime.InteropServices;
+using cafeRecAPI.Service;
+
+namespace cafeRecAPI.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class CafeApiController : ControllerBase
+    {
+        private readonly ILogger<CafeApiController> _logger;
+        private readonly ICafeService cafeService;
```

Comment:
Good standards to prefix private fields with underscore
```suggestion
        private readonly ICafeService _cafeService;
```